app/helpers/application_helper.rb

```diff
@@ -0,0 +1,18 @@
+module ApplicationHelper
+  def sortable(column, title = nil)
+    title ||= column.titleize
+    if css_class = column == sort_column
+      "current #{sort_direction}"
+    else
+      nil
+    end
+    if direction = column == sort_column && sort_direction ==
+      "asc"
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.